content/posts/2022-09-26-bundle-option-in-crc.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Different bundle types option in crc"
+date: 2022-09-26T10:41:10+05:30
+---
+
+Until 2.9 release CRC bundle config option is only able to accept local bundle path
+which user needs to download before consuming it. It has been around 3 releases since
+podman and okd bundles are migrated to quay instead of mirror location and team was
+brainstorming around how user can now download those bundles before consuming if that's
+only option.
+
+After some team discussions we all agreed to extend the `bundle` config option to take
+either a registry url, http/https location or local path (to make sure backward compatibility).
+
+With CRC 2.9.0 release now bundle config option support following cases and should work as expected.
```

Comment:
Starting with CRC 2.9.0 `bundle` config option supports three different ways to configure the bundle location, these are explained with three example use cases below: